pkgs/development/tools/database/sqlboiler-crdb/default.nix

```diff
@@ -0,0 +1,25 @@
+{ buildGoModule, fetchFromGitHub, lib }:
+
+buildGoModule rec {
+  pname = "sqlboiler-crbd";
+  version = "unstable-2022-06-12";
+
+  src = fetchFromGitHub {
+    owner = "glerchundi";
+    repo = "sqlboiler-crdb";
+    rev = version;
```

Comment:
Oh I'm sorry, this won't work. You have to put the commit hash manually. If you have tested this locally, it has evaluated because the `sha256` is the same.